pkg/operator/k8sutil/network.go

```diff
@@ -77,13 +79,13 @@ func ApplyMultus(net rookv1.NetworkSpec, objectMeta *metav1.ObjectMeta) error {
 			shortSyntax = true
 		}
 
-		isCsi := strings.Contains(objectMeta.Labels["app"], "csi-")
-		if isCsi {
+		isOSD := strings.Contains(objectMeta.Labels["app"], "osd")
```

Comment:
Let's not hard-code the apps that need to be excluded. How about defining a slide of the apps to exclude, then this implementation can iterate over the slice. Otherwise, we have to add every exception as a new line of code instead of just adding to the slice when there is a new exception.